fix(release): Update SDKs to use API version 2021-03-30

SunithaGudisagarIBM opened the pull request:
NEW FEATURES

Introduced console access token for an instance
Update, Retrieve an instance disk and List all instance disks
Update, Retrieve a dedicated host disk and List all disks on dedicated host
Body and Query is introduced in type of rules in Load Balancer Listener Policy Rule
Introduced Profile in instance patch
Pagination support for Instance Group Policies and Membership

sumitkumartiwari reviewed (approved):
Merge request comment has been correctly updated for release.

SunithaGudisagarIBM commented:
NEW FEATURES

Introduced console access token for an instance
Update, Retrieve an instance disk and List all instance disks
Update, Retrieve a dedicated host disk and List all disks on dedicated host
Body and Query is introduced in type of rules in Load Balancer Listener Policy Rule
Introduced Profile in instance patch
Pagination support for Instance Group Policies and Membership

ibm-vpc commented:
:tada: This PR is included in version 1.0.1 :tada:

The release is available on [GitHub release](https://github.com/IBM/vpc-go-sdk/releases/tag/v1.0.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
